perfkitbenchmarker/linux_packages/redis_server.py

```diff
@@ -17,8 +17,8 @@
 
 from perfkitbenchmarker import vm_util
 
-REDIS_TAR = 'redis-2.8.9.tar.gz'
-REDIS_DIR = '%s/redis-2.8.9' % vm_util.VM_TMP_DIR
+REDIS_TAR = 'redis-stable.tar.gz'
```

Comment:
Its better to target this to a specific release.  We've had issues with taking "latest, stable, etc".